Compartilhando implementação do pdfplumber

maikoademirnunesti opened the pull request:
Compartilhando a implementação do pdfplumber para conversão de PDF sem imagem e sem gráficos em texto puro TXT.

chiru-stefan commented on Extracao_de_dados/via_pdf/heuristics/is_scanned_pdf.py:
```diff
@@ -0,0 +1,29 @@
+    with pdfplumber.open(pdf_path) as pdf:  # Abre o arquivo PDF especificado pelo caminho usando pdfplumber.
+        for page in pdf.pages:  # Itera por todas as páginas do PDF.
+            if page.images:  # what is happening if the page has an image in the header?
+                # what if the image contains a graph as an image
```
future development: add [Tesseract](https://github.com/tesseract-ocr/tesseract)

maikoademirnunesti replied:
Eu vou implementar sua sugestão.

maikoademirnunesti committed:
Compartilhando implementação do pdfplumber

Compartilhando a implementação do pdfplumber para conversão de PDF sem imagem e sem gráficos em texto puro TXT.
